<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小车调试界面</string>
  </property>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>600</y>
     <width>89</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="cameraView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>581</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDisplay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>小车远程图像显示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>小车IP地址</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditIP">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideoPort">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>视频端口号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditVideoPort">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelControlPort">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>控制端口号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditControlPort">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>89</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>状态信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="smartCarLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="carLeft">
      <property name="text">
       <string>左</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="carRight">
      <property name="text">
       <string>右</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="carUp">
      <property name="text">
       <string>上</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="carDown">
      <property name="text">
       <string>下</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelControl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>小车控制</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="steerLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="steerLeft">
      <property name="text">
       <string>左</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="steerRight">
      <property name="text">
       <string>右</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="steerUp">
      <property name="text">
       <string>上</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="steerDown">
      <property name="text">
       <string>下</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelSteer">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>舵机控制</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserStatus">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonScreenSpot">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>300</y>
     <width>89</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>截图</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDriveModel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>驾驶模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonHand">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>500</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>手动驾驶</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAutomatic">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>460</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>自动驾驶</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWheelSpeed">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>速度大小</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWheelDelay">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>电机延时</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDisplay">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>显示信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDebugModel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>调试模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSpeed">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDelay">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
